### DISCUSSION TITLE
Sonoff Dual R3 (dual switch only)

### DISCUSSION THREAD

**Discussion by nickforev** [RavenSystem/esp-homekit-devices#1993] [MEPLHAA Scripts Collection]
o Device Name: Sonoff Dual R3
o URL to Device: https://sonoff.tech/product/diy-smart-switches/dualr3/
o Used HAA version: 12.0.0
o Summary of functions: Dual switch only
O MEPLHAA script: {"c":{"io":[[[13],3],[[0,32,33],6],[[14,27],2]],"l":13,"b":[{"g":0,"t":5}]},"a":[{"0":{"r":[{"g":27}]},"1":{"r":[{"g":27,"v":1}]},"b":[{"g":0,"t":1},{"g":32,"t":0}],"s":5},{"0":{"r":[{"g":14}]},"1":{"r":[{"g":14,"v":1}]},"b":[{"g":33,"t":0}],"s":5}]}
o Long description: Last state option, only switch without other factory functions, power meter don't work.
• Photos, additional information:![image](https://user-images.githubusercontent.com/67782540/234506615-59402b35-4cb5-44fa-8f39-508fde24fc8b.jpeg)

**Comment by kivanc87** [comment]


**Reply by nickforev** [reply]
Can i help you?

**Reply by nickforev** [reply]
This discussion about Sonoff Dual R3.
Try search scrypt in device database.

**Comment by RavenSystem** [comment]
https://github.com/RavenSystem/esp-homekit-devices/wiki/Sonoff-Dual-R3-v1.6
